dev/reference/supported-clients.md

```diff
@@ -56,6 +56,7 @@ Oracle 官方提供了以下 API，TiDB 可以兼容所有这些 API。
 | Haskell        | Haskell MySQL Bindings                   | Native Driver                    | See [Brian O'Sullivan's pure Haskell MySQL bindings](http://www.serpentine.com/blog/software/mysql/). |
 |                | `hsql-mysql`                             | `libmysqlclient`                 | See [MySQL driver for Haskell](http://hackage.haskell.org/cgi-bin/hackage-scripts/package/hsql-mysql-1.7). |
 | Java/JDBC      | Connector/J                              | Native Driver                    | See [MySQL Connector/J 5.1 Developer Guide](https://dev.mysql.com/doc/connector-j/5.1/en/). |
+| Kaya           | MyDB                                     | `libmysqlclient`                 | See [MyDB](http://kayalang.org/library/latest/MyDB). |
```

Comment:
This row was removed in https://github.com/pingcap/docs-cn/pull/2218.

Reply:
I think it's because my [forked repo](https://github.com/yikeke/docs-cn) is 2 commits ahead of pingcap:master. I'll resolve this issue now.

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/2235/commits/67bdf8225ac56df3eb3efe2b813769af682f0e6b